k8s/sandbox/common/idam-sprod/idam-api.yaml

```
@@ -0,0 +1,40 @@
+---
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: idam-api
+  namespace: idam-sprod
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:preview-*
+spec:
+  releaseName: idam-api
+  rollback:
+    enable: true
+    retry: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: idam-api
+    version: 0.2.4
+  values:
+    java:
+      image: hmctspublic.azurecr.io/idam/api:preview
+      ingressHost: idam-api-sbox.service.core-compute-sprod.internal
```

```suggestion
      ingressHost: idam-api-sprod-sbox.service.core-compute-sandbox.internal
```

This requires additional listeners on the backend application gateway https://github.com/hmcts/azure-platform-terraform/blob/master/environments/sbox/backend_lb_config.yaml#L59-L64

---

https://github.com/hmcts/azure-platform-terraform/pull/154

---

```suggestion
      ingressHost: idam-api-sprod-sbox.service.core-compute-sandbox.internal
```
This still needs to be as suggested. https://github.com/hmcts/azure-platform-terraform/pull/154 will construct `idam-sprod-api-sbox.service.core-compute-sandbox.internal` from the configuration you have supplied

---

I would suggest you create a new PR for hmcts/azure-platform-terraform with 

```
      - product: idam
        component: api-sprod
        cookie_based_affinity: Enabled
      - product: idam
        component: web-admin-sprod
        cookie_based_affinity: Enabled
```